server/migrations/20211117015845-rank-search-results.js

```diff
@@ -0,0 +1,30 @@
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.transaction(async (t) => {
+
+      await queryInterface.sequelize.query(
+        'UPDATE "OffchainThreads" SET _search = ' +
+        '(setweight(to_tsvector(\'english\', coalesce(title, \'\')), \'A\') || ' +
```

Comment:
For my clarity, how is this vector being constructed? And we're certain the vector definition doesn't need to be made on the typed model itself?

Reply:
@jnaviask

Reply:
The vector's been in place since Search 1.0, migration: `server/migrations/20201215074113-add-search-indexes.js`. A type gets set there, here we're calling the same function (`to_tsvector`) with some custom weights set up.